components/gfx/font_cache_thread.rs

```diff
@@ -216,7 +216,8 @@ impl FontCache {
                     url: url.clone(),
                     type_: RequestType::Font,
                     destination: Destination::Font,
-                    origin: url.clone(),
+                    // TODO: Add a proper origin - Can't import GlobalScope from gfx
```

Comment:
I meant something like adding a new `origin` field to the `AddWebFont` enum tuple variant, and then look at all the places where `AddWebFont` is created/called, and add the appropriate `Origin` for it. Does that make sense?